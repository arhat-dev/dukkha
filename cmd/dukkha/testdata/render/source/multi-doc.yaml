foo@env: GLOBAL_ENV_FOO=${GLOBAL_ENV_FOO}
bar@shell: template:archconv.DebianArch \"amd64\"
woo@template:
- hello {{ archconv.AlpineArch "arm64" }}
str_not_number@transform:
  value@template: "{{- .Values.testdata.str_not_number -}}"
---
a@echo|env: ${Values.testdata.render.a}
b@template: "{{ .Values.testdata.render.b | toYaml }}"
---
- 1
- 2
- foo@env:
    bar@env: GLOBAL_ENV_BAR=${GLOBAL_ENV_BAR}
    woo@echo: nested rendering
