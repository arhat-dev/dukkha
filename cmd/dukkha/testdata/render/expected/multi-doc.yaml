bar: amd64
foo: GLOBAL_ENV_FOO=foo
str_not_number: str-10.10000
woo:
  - hello aarch64
---
a: I'm a
b:
  c:
    d:
      e: echo "Hello from gist.github.com"
---
- 1
- 2
- foo:
    bar: GLOBAL_ENV_BAR=foo-bar
    woo: nested rendering
